### PR TITLE
feat(kps): Upgrade to 56.0.3

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

feat

**What this PR does/ why we need it**:

Upgrade kps to latest, and add a few more files to .helmignore ... installing the helm chart was failing with ` Helm install failed: create: failed to create: Secret "sh.helm.release.v1.kube-prometheus-stack.v1" is invalid: data: Too long: must have at most 1048576 bytes` :/

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by takirala — APPROVED** [review]

File: staging/kube-prometheus-stack/.helmignore [hunk 1]
```diff
@@ -30,3 +30,5 @@ unittests/
 upgrade_operator.sh
 kube-prometheus-stack-*.tgz
 patch/
+CONTRIBUTING.md
+UPGRADING.md
```
Comment (msdolbey) [inline staging/kube-prometheus-stack/.helmignore:34]: I don't see `UPGRADING.md` in this branch or in master - does that matter?

File: staging/kube-prometheus-stack/patch/patch_9_mesosphere_helmignore.sh [hunk 2]
```diff
@@ -15,6 +15,8 @@ cat << EOF >> ${SRCFILE}
 upgrade_operator.sh
 kube-prometheus-stack-*.tgz
 patch/
+CONTRIBUTING.md
+UPGRADING.md
```
Comment (msdolbey) [inline staging/kube-prometheus-stack/patch/patch_9_mesosphere_helmignore.sh:19]: same - I don't see `UPGRADING.md` in this branch or in master - does that matter?

File: staging/kube-prometheus-stack/.helmignore (hunk 1 above)
Comment (gracedo) [reply staging/kube-prometheus-stack/.helmignore:34]: https://github.com/mesosphere/charts/blob/master/staging/kube-prometheus-stack/UPGRADING.md

Comment (msdolbey) [reply staging/kube-prometheus-stack/.helmignore:34]: ahhhhhhhh
gooootcha. My error.

Comment (msdolbey) [reply staging/kube-prometheus-stack/.helmignore:34]: I mean - I knew it had to be somewhere. I just didn't know where.

**Pull request merged**